appium/tests/data/index.ts

```diff
@@ -213,7 +213,12 @@ export const CommonData = {
     email: 'revoked@domain.test',
     name: 'revoekd@flowcrypt.com'
   },
-  keyManagerURL : {
+  refreshKeys: {
+    wrongPassPhrase: 'Error\nIncorrect pass phrase. Please try again.',
+    updatedSuccessfully: 'Account keys updated',
+    errorMessage: 'Error\nCould not update keys from EKM due to error: '
```

Comment:
After some thought, let's change to `Error updating account keys: `

Reply:
ok